vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -157,9 +157,39 @@ void ImageViewManager::EmitImageEvent(
   reactInstance->DispatchEvent(tag, eventName, std::move(eventData));
 }
 
+winrt::Size ImageViewManager::getMaxSize(const folly::dynamic &reactDiffMap) {
+  float width{0};
+  float height{0};
+
+  width = tryGetPropAsFloat(reactDiffMap, "maxWidth");
+  if (width == 0) {
+    width = tryGetPropAsFloat(reactDiffMap, "width");
+  }
+
+  height = tryGetPropAsFloat(reactDiffMap, "maxHeight");
+  if (height == 0) {
+    height = tryGetPropAsFloat(reactDiffMap, "height");
+  }
+
+  return {width, height};
```

Comment:
>return {width, height}; [](start = 2, length = 23)

Question:  What size will LoadedImageSurface decode to if one or both dimensions come out 0?  I seem to remember it will decode to natural size if either dimension is 0.  Is that correct?

Reply:
Yes, it will decode to natural size if either dimension is 0.